docs/source/notebooks_and_ipython/add_kedro_to_a_notebook.ipynb

```diff
@@ -0,0 +1,896 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e2dbc66a",
+   "metadata": {},
+   "source": [
+    "# Add Kedro features to a notebook\n",
+    "\n",
+    "This page describes how to add Kedro features incrementally to a notebook. \n",
+    "\n",
+    "It starts with a version of the spaceflights example with does NOT use Kedro that you can run inside a notebook. The example converts portions of the code to use Kedro features while remaining runnable from within a notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d85737b7",
+   "metadata": {},
+   "source": [
+    "## Spaceflights in a notebook\n",
+    "\n",
+    "If you are unfamiliar with the spaceflights example, it is used to introduce the basics of Kedro in a tutorial that runs exclusively as a Kedro project, that is, as a set of `.py` files rather than in a notebook. The premise is as follows:\n",
+    "\n",
+    "_It is 2160, and the space tourism industry is booming. Globally, thousands of space shuttle companies take tourists to the Moon and back. You have been able to source data that lists the amenities offered in each space shuttle, customer reviews, and company information._\n",
+    "\n",
+    "_Project: You want to construct a model that predicts the price for each trip to the Moon and the corresponding return flight._\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f406dc0",
+   "metadata": {},
+   "source": [
+    "### The notebook example\n",
+    "The full example code is given below. In a real notebook example, it would most likely be split across multiple cells with various additional code added to inspect and explore the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7cca8e81-e49c-4c75-b80b-e27b2c61682f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "# This code needs the data folder set up and three files downloaded. \n",
+    "# Is there a way to do this in code from the notebook to save the reader the manual task?\n",
+    "# Either download the file or use an OS data fabricator to make 3 files?\n",
+    "\n",
+    "companies = pd.read_csv('data/companies.csv')\n",
+    "reviews = pd.read_csv('data/reviews.csv')\n",
+    "shuttles = pd.read_excel('data/shuttles.xlsx', engine='openpyxl')\n",
```

Comment:
Indeed, the notebook execution fails at this point because the files don't exist.

Welcome to a new beautiful Sphinx stack trace: https://beta.readthedocs.org/projects/kedro/builds/22122501/#205249990--1458

Reply:
Oh curses. I don't want to put them in the docs folder really, so tomorrow I'll set up an examples subfolder somewhere for this...unless you have any thoughts on where/how to do this (build them on the fly?)

Reply:
@astrojuanlu For now, it's on my github pages: https://stichbury.github.io/notebooks_and_ipython/kedro_and_notebooks.html

Reply:
@astrojuanlu [I put it up on GitHub pages](https://stichbury.github.io/notebooks_and_ipython/add_kedro_to_a_notebook.html)